### DISCUSSION TITLE
Questions on Attribution Reporting API Origin Trial

### DISCUSSION THREAD

**Discussion by lbdvt** [privacysandbox/privacy-sandbox-dev-support#22] [Replacing cross-site tracking] [answered]
Hi,

Could you please confirm that the components of the Attribution Reporting API that are available as of today for Origin Trial are those listed here:
https://developer.chrome.com/docs/privacy-sandbox/attribution-reporting-introduction/#status ?

In that table, "Attribution model: priority-based Explainer" links to [Attribution Reporting for View-Through Measurement](https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting_views.md#controlling-which-attribution-source-to-triggerd).
Could you please confirm that it should actually link to: [Multiple sources for the same trigger (Multi-touch)](https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting_clicks.md#multiple-sources-for-the-same-trigger-multi-touch) ?

The corresponding OT is https://developer.chrome.com/origintrials/#/view_trial/3411476717733150721 ?

And lastly, is there any restriction on the OT (e.g. geo, like there was for FLoC OT) ?

Thanks!

**Comment by lbdvt** [comment]
@maudnals maybe?

Thanks

**Reply by maudnals** [reply]
Hi @lbdvt,
Thanks for your questions and the ping!
I've replied below.

**Comment by maudnals** [comment, marked as answer]
Hi @lbdvt,

Thanks for your questions!

> Could you please confirm that the components of the Attribution Reporting API that are available as of today for Origin Trial are those listed here:
https://developer.chrome.com/docs/privacy-sandbox/attribution-reporting-introduction/#status ?

The components of the Attribution Reporting API that are available as of today for Origin Trial are those listed in that **Status** list and that have a status of `🧪 Experiment`.

> In that table, "Attribution model: priority-based Explainer" links to Attribution Reporting for View-Through Measurement.
Could you please confirm that it should actually link to: Multiple sources for the same trigger (Multi-touch) ?

Indeed, "Attribution model: priority-based Explainer" could now link to [Multiple sources for the same trigger](https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting_clicks.md#multiple-sources-for-the-same-trigger-multi-touch). We'll fix this, thanks for flagging!

> The corresponding OT is https://developer.chrome.com/origintrials/#/view_trial/3411476717733150721 ?

Yes, you can use source-side priorities to define custom attribution rules in the currently running OT.

> And lastly, is there any restriction on the OT (e.g. geo, like there was for FLoC OT) ?

There is no geo restriction for this OT, and no additional restriction. 

**💡 Noteworthy:**
* If you're looking for a quick way to see how the priority feature behaves, **head over to the [deployed demo](https://attribution-reporting-demo.glitch.me/)** and corresponding code that's linked from the demo page. The demo uses priorities in a basic way, to define a first-click attribution model.
* **Check out the [newly published API guide](https://developer.chrome.com/docs/privacy-sandbox/attribution-reporting-event-guide/#define-custom-attribution-rules)** for an explanation of how to use the priority feature (and all API features).
* You can use priorities by leveraging the currently running OT, but you can also see it in action by just turning on flags in your browser to activate the API locally; this is handy for local experimentation. [See how](https://developer.chrome.com/docs/privacy-sandbox/attribution-reporting-event-guide/#enable-the-api-in-your-browser). (This is also how the demo linked above works)
* Just in case you're not already in there, feel free to join the [mailing list for devs](https://groups.google.com/u/0/a/chromium.org/g/attribution-reporting-api-dev). [EDIT: The reason I'm suggesting this is that we use that list as one of the first places to post developers updates, including new API guides like the one I've linked above; so in case that API guide is interesting to you, join the mailing list if you wish to be notified as soon as we publish such guides.]

Let me know if you have any follow-up questions!

**Reply by lbdvt** [reply]
Very clear, thanks!
